fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -215,7 +220,16 @@ protected void configureConnectionPool() {
      */
     protected void buildCommonModel(PhysicalDataModel pdm, boolean fhirSchema, boolean oauthSchema, boolean javaBatchSchema) {
         if (fhirSchema) {
-            FhirSchemaGenerator gen = new FhirSchemaGenerator(schema.getAdminSchemaName(), schema.getSchemaName(), isMultitenant());
+            String resourceTypesString = properties.getProperty("resourceTypes");
+
+            FhirSchemaGenerator gen;
+            if (resourceTypesString == null || resourceTypesString.isEmpty()) {
+                gen = new FhirSchemaGenerator(schema.getAdminSchemaName(), schema.getSchemaName(), isMultitenant());
+            } else {
+                Set<String> resourceTypes = new HashSet<>(Arrays.asList(resourceTypesString.split(",")));
```

Comment:
You could use .stream().filter(x -> ModelSupport.isResourceType(x)).collect(Collection.toList())

Reply:
we want a set, not a list.  we could use streams and Collection.toSet() but I think its equivalent.  any reason you think the stream version is better or just a style preference?

Reply:
oh i see you are proposing to validate the input here as well, but I made the decision to pass whatever the user passes to us on and I do the resourceType validation right in FhirSchemaGenerator.
specifically, I decided to generate tables for whatever they pass us (but log a warning if its not a known resource type).
if we want to reverse that decision, then we should at least log a warning if we're gonna ignore the input.
the option i considered is to just throw if they pass an invalid resource type name.